src/OrbitGl/LiveFunctionsController.cpp

```diff
@@ -217,12 +217,18 @@ uint64_t LiveFunctionsController::GetStartTime(uint64_t index) {
   return GetCaptureMin();
 }
 
-uint64_t LiveFunctionsController::GetCaptureMin() { return GCurrentTimeGraph->GetCaptureMin(); }
-uint64_t LiveFunctionsController::GetCaptureMax() { return GCurrentTimeGraph->GetCaptureMax(); }
+uint64_t LiveFunctionsController::GetCaptureMin() {
```

Comment:
const and [[nodiscard]]?

Reply:
Good catch. I can do a follow-up PR on Monday. There were quite a bit of clang-tidy warnings in this file.